test.js

```diff
@@ -32,3 +32,33 @@ describe('repos', () => {
     expect(scopedNames.length).to.be.above(32 * 1000)
   })
 })
+
+describe('stats', () => {
+  const fs = require('fs')
+  const path = require('path')
+  const tpl = require('./lib/stats-tpl')
+
+  it('sould match the readme table', () => {
+    const readme = fs.readFileSync(path.join(__dirname, '/readme.md'))
+    expect(readme).to.match(tpl.regex)
+  })
+
+  it('should build the correct table', () => {
+    const metadata = {
+      packages: 12,
+      repos: {
+        foo: 1,
+        bar: 2,
+        baz: 6
+      }
+    }
+    const table = tpl.build(metadata)
+    expect(table)
+      .to.be.a('string')
+      .to.match(tpl.regex)
+      .to.match(/With repository in package.json \| 12 \| 100%/)
+      .to.match(/Foo \| 1 \| \d+\.\d{2}%/)
+      .to.match(/Bar \| 2 \| \d+\.\d{2}%/)
+      .to.match(/Baz \| 6 \| \d+\.\d{2}%/)
```

Comment:
please add the expected percent, so we can validate numbers are correct.

You can use factors of `3` and `4`, which divided by `12` would lead to nice numbers..

- `3/12 == 25.00%`
- `6/12 == 50.00%`
- `9/12 == 75.00%`
- `4/12 == 1/3 == 33.33%`
- `8/12 == 2/3 == 66.66%`